_includes/_wallet_install.htm

```diff
@@ -6,55 +6,29 @@
                 <h4 class="text-bold">
                     Install the Gridcoin client
                 </h4>
-                <h5>
-                    Microsoft Windows
-                </h5>
-                <ul>
-                    <li>
-                        <a class="btn btn-round grcbtn" href="https://github.com/gridcoin-community/Gridcoin-Research/releases/latest">Download Windows Client</a>
-                    </li>
-                </ul>
-            </div>
-            <div class="col-12 col-sm-12 col-md-12">
-                <h5>
-                    Mac OS
-                </h5>
-                    <a class="btn btn-round grcbtn" href="https://github.com/Git-Jiro/homebrew-jiro/releases">
-                        DMG packages
-                    </a>
-                    <a class="btn btn-round grcbtn" href="http://wiki.gridcoin.us/OS_X_Guide">Gridcoin wiki: Mac OS guide</a>
-            </div>
-            <div class="col-12 col-sm-12 col-md-12">
-                <h5>
-                    Debian Packages (Ubuntu)
-                </h5>
-                <a class="btn btn-round grcbtn" href="https://code.launchpad.net/~gridcoin/+archive/ubuntu/gridcoin-stable">Launchpad built ubuntu releases</a>
-                <ul>
-                    <li>sudo add-apt-repository ppa:gridcoin/gridcoin-stable</li>
-                    <li>sudo apt-get update</li>
-                    <li>sudo apt-get install gridcoinresearch-qt</li>
-                    <li>gridcoinresearch</li>
-                </ul>
-            </div>
-            <div class="col-12 col-sm-12 col-md-12">
-                <h5>
-                    RPM Packages (OpenSUSE/Fedora)
-                </h5>
-                <a class="btn btn-round grcbtn" href="https://software.opensuse.org/package/gridcoinresearch">OpenSuse/Fedora RPMs</a>
-            </div>
-            <div class="col-12 col-sm-12 col-md-12">
-                <h5>
-                    AUR (Arch Linux)
-                </h5>
-                <a class="btn btn-round grcbtn" href="https://aur.archlinux.org/packages/gridcoinresearch-qt">Archlinux AUR package</a>
-            </div>
-            <div class="col-12 col-sm-12 col-md-12">
-                <h5>
-                    Manually compile the Gridcoin client
-                </h5>
-                <a class="btn btn-round grcbtn" href="http://wiki.gridcoin.us/Linux_guide">Linux compiling guide</a>
-                <a class="btn btn-round grcbtn" href="https://github.com/gridcoin/Gridcoin-Research">Gridcoin GitHub Repo</a>
-            </div>
+                <h4>Windows & Linux</h4>
```

Comment:
Could you reinstate then col's and <div>s for formatting here, Looks all scrunched up right now.

Reply:
I think putting the formatting like the index page has MacOS being too far out meaning that it can be easily missed 
![image](https://user-images.githubusercontent.com/30132912/100936100-00b21200-34bf-11eb-89e4-709af8f34268.png)

Tried playing around with it (putting the MacOS column closer, spiting it differently, etc) and I feel like what I put in the PR was the best I could get from that. I'm sure there's some better way to do it, but I couldn't find it